### ISSUE TITLE
Show only editable sites

### ISSUE THREAD

**Issue by bar-tech-dev** [solspace/craft3-calendar#224]
**Describe the bug or issue you're experiencing**
CP calendar overview shold only list sites where user has privileges to edit.

**Steps to reproduce**

1. Create multisitite Craft installation
2. Add one user with permission to edit only one of them, and second user with permission to edit second one.

**Expected behavior**
Users should see only events/sites they have privileges to edit.

**Craft & Plugin Info (please complete the following information):**

- Craft Version: 4.4
- Calendar Version: 4.0
- Calendar Edition: free
- Fresh Install or Upgrade: fresh

**Additional context**
![image](https://user-images.githubusercontent.com/72630492/229912587-4983cc37-7c37-489a-9cd1-35385d3d70e2.png)

It can be done by add following change:
```
        $user = Craft::$app->getUser()->getIdentity();
        $siteMap = [];
        if (\Craft::$app->getIsMultiSite()) {
            foreach (\Craft::$app->sites->getAllSites() as $site) {

                // check permissions
                if(!$user->can("editSite:".$site->uid)){
                    continue;
                }

                if (!\in_array($site->id, $enabledSiteIds)) {
                    continue;
                }

                if ($site->id === $currentSiteId) {
                    $selectedSiteId = $currentSiteId;
                }

                $siteMap[$site->id] = $site->name;
            }
        }
```

**Comment by kjmartens** [comment]
Thanks for this @bartutt. I'll have a developer check into it. 🙂

**Comment by kjmartens** [comment]
This is now fixed in Calendar 4.0.9. 🙂
